app/src/main/java/com/duckduckgo/app/cta/ui/CtaViewModel.kt

```diff
@@ -111,6 +111,12 @@ class CtaViewModel @Inject constructor(
         }
     }
 
+    fun onUserClickCtaSecondaryButton(cta: SecondaryButtonCta) {
+        cta.secondaryButtonPixel?.let {
+            pixel.fire(it, cta.pixelSecondaryButtonParameters())
```

Comment:
is this ok being launched in the main thread?

Reply:
`ApiBasedPixel`:`Pixel` will excecute it in a background thread. So we are good here.